### DISCUSSION TITLE
Which is the last compatible version with Windows 7 ?

### DISCUSSION THREAD

**Discussion by apprenti-sorcier** [edrlab/thorium-reader#2674] [General]
I read on the website that Thorium requires now Windows 10. What about Windows 7, which version should I pick?
I couldn’t find it in the change log, as well as any information about Electron.

**Comment by danielweck** [comment]
Some information here: https://www.electronjs.org/blog/windows-7-to-8-1-deprecation-notice

**Reply by apprenti-sorcier** [reply]
That link is about Electron and I don’t even know what that is.
My question is again about Thorium: Which version should I use that is the last compatible version with Windows 7?

**Reply by danielweck** [reply]
Sorry for not being clear :)
Thorium is an Electron application, therefore the minimum Operating System requirements are dictated by the Electron software project. Thorium is at Electron version 30+ already, way past the cut-off date for Windows 7 support (as documented in their blog post)

**Comment by apprenti-sorcier** [comment]
That I have already read. But older versions are available on Github and obviously I’m not looking for the latest version published, but the latest version compatible, be it 1.2.3 or 2.3.4.

**Reply by danielweck** [reply]
"Electron 22 will be the last Electron major version to support Windows versions older than 10. Windows 7/8/8.1 will not be supported in Electron 23 and later major releases."

Thorium 2.2.0 = Electron 21.3.1
https://github.com/edrlab/thorium-reader/blob/v2.2.0/package.json#L354

...next Thorium version in the release history:

Thorium 2.3.0 = Electron 25.4.0
https://github.com/edrlab/thorium-reader/blob/v2.3.0/package.json#L358

So, it looks like Thorium 2.2.0 is the last official version that supported Windows 7

**Reply by apprenti-sorcier** [reply]
Thank you @danielweck for this information. It has been very helpful.
